de-lib/src/main/java/org/iplantc/de/admin/desktop/client/toolRequest/view/ToolRequestViewImpl.java

```diff
@@ -64,6 +68,8 @@ public ToolRequestViewImpl(final ToolRequestProperties trProps,
         initWidget(uiBinder.createAndBindUi(this));
         grid.getSelectionModel().addSelectionChangedHandler(this);
         grid.getSelectionModel().setSelectionMode(SelectionMode.SINGLE);
+        //show latest tool request first
+        store.addSortInfo(new Store.StoreSortInfo<ToolRequest>(trProps.dateSubmitted(), SortDir.DESC));
```

Comment:
Note that the `tool-requests` endpoints support paging, so if the UI takes a while to load while sorting these results in the future, it might be faster to let the endpoint do the sorting.